aws/windows-agent/main.tf

```diff
@@ -29,12 +29,26 @@ module "dcos" {
   bootstrap_instance_type = "m4.xlarge"
 
   dcos_variant              = "ee"
-  dcos_version              = "1.13.1"
+  dcos_version              = "2.1.0"
   dcos_license_key_contents = "${file("~/license.txt")}"
+  ansible_bundled_container = "mesosphere/dcos-ansible-bundle:windows-beta-support"
+
+  #Linux Installer path - place url with "pull/PR#" or "master" suffix here:
+  custom_dcos_download_path = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh"
```

Comment:
I see the point here but for a tested version of this example with a stable result an immutable release should be chosen. E.g. a commit-hash which was verified to work with this example. 

If this stay with master results will be different every time this example is ran by someone

Reply:
for now master artifact is good enough.